internal/service/environment.go

```diff
@@ -23,3 +40,36 @@ func (e *Environment) ListEnvironments(ctx *context.Context, request *environmen
 
 	return response, nil
 }
+
+// CreateEnvironment creates environment
+func (e *Environment) CreateEnvironment(ctx *context.Context, request *environment.CreateEnvironmentRequest) error {
+	conn, requestCtx, err := grpcClient(ctx)
+	if err != nil {
+		return err
+	}
+	client := environment.NewEnvironmentServiceClient(conn)
+	stream, err := client.CreateEnvironment(*requestCtx, request)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Creating environment...")
+	spinner := spinner.New(spinner.CharSets[SpinnerType], SpinnerDelay)
+	spinner.Color(SpinnerColor, SpinnerStyle)
+
+	for {
+		response, err := stream.Recv()
+		spinner.Stop()
+		if err != nil {
```

Comment:
We should catch the grpc connection closed errors and retry automatically instead of asking user to retry

Reply:
Need to identify error types which should be retried automatically.
Will address in separate PR